CHANGELOG.md

```diff
@@ -15,6 +15,9 @@ x.x.x Release notes (yyyy-MM-dd)
   cannot exist independently from a `Realm`.
 * Aggregate operations are now available on `List`: `min`, `max`, `sum`,
   `average`.
+* Committing write transactions (via `commitWrite` / `commitWriteTransaction` and 
+  `write` / `transactionWithBlock`) now optionally allow for handling errors when
```

Comment:
actually, thinking about this further, we haven't typically included Swift 2-specific changes in the changelog, so it's fine to keep as-is.